### ISSUE TITLE
Lead Sponsor Footer Block

### ISSUE THREAD

**Issue by willykaram** [NYC-Camp/website#101]
We need to create a block for sponsors that will appear that the bottom of every page on the NYC Camp site and will persistently display the logos of our lead sponsors. This block will be static until a users scroll to the very bottom of the page, and then the block will start to scroll up the page in order to display the rest of the sponsors.

By keeping the footer block static until a user scrolls to the bottom of the page.

The footer blocks that appear when you scroll up would be similar to those on the Capital Camp (http://capitalcamp.org/) site or the Prague site (https://prague2013.drupal.org/node), except the logos on our page would get bigger for higher levels of sponsorship. The blocks used on the sponsors page on the 2013 NYC Camp site should serve as a solid basis for this work: http://2013.nyccamp.org/sponsors

**Comment by jmarkel** [comment]
Working on the Sponsorship footer block. The existing views need changing since the underlying data architecture has changed a bit.

**Comment by jmarkel** [comment]
The Sponsor content type has 2 fields, both called Sponsorship level, neither of which seem to be needed any longer. Instead I am adding an Entity Ref field that refers to the Sponsorship Option content type - and that field will be used in the view in place of the sponsorship level

**Comment by willykaram** [comment]
@jmarkel We are using page manager to create the page at http://nyccamp.org/sponsors, the page can be edited here: 

http://nyccamp.org/admin/structure/pages/nojs/operation/page-sponsors/handlers/page_sponsors_panel_context/content

It is composed of the various views blocks for sponsors

**Comment by willykaram** [comment]
You can see how this is currently implemented at http://2013.nyccamp.org

**Comment by willykaram** [comment]
These are the view blocks for the different sponsorship levels:

http://nyccamp.org/admin/structure/views/view/sponsors_lead/edit

http://nyccamp.org/admin/structure/views/view/sponsors_core/edit

http://nyccamp.org/admin/structure/views/view/sponsors_contributing/edit

http://nyccamp.org/admin/structure/views/view/sponsors_supporting/edit

http://nyccamp.org/admin/structure/views/view/sponsors_community/edit

http://nyccamp.org/admin/structure/views/view/sponsors_summits/edit

http://nyccamp.org/admin/structure/views/view/sponsors_sprints/edit

**Comment by willykaram** [comment]
We should ideally be able to use or clone the existing Lead Sponsors block and the bulk of the work relates to theming it to be a static footer.

**Comment by willykaram** [comment]
Any updates on how we're making out on this. Is it okay if I tackle this if we're stalled? We now need to get sponsors up on the homepage footer.

**Comment by rabellamy** [comment]
@willykaram as of now http://nyccamp.org/admin/structure/views/view/sponsors_lead/edit is only returning one result. Do you envision more? I have mocked up how it could look with more results by borrowing the 'Partners Spotlight' block. You can see it here:
http://leadsponsor-nyccamp2014.gotpantheon.com/welcome
and here:
http://leadsponsor-nyccamp2014.gotpantheon.com/team

**Comment by willykaram** [comment]
@rabellamy What we need is just a feature for lead, sponsors, I've changed this on your multidev, so that's the block that now appears. If someone else can comment that this is okay then I'll make a similar change on test/live. http://leadsponsor-nyccamp2014.gotpantheon.com

**Comment by mirie** [comment]
Good to go -- looks like /partner and /partners don't have that footer block. I'm assuming that this is intentional -- anyways, :horse:

**Comment by Offlein** [comment]
Seems legit.

**Comment by rabellamy** [comment]
I will create a pull request in the morning.

**Comment by willykaram** [comment]
@rabellamy we shouldn't create a bull request for what you coded. See my comment above from yesterday. What we need it a bit different. I am going to implement based on Craig's comment above.

**Comment by rabellamy** [comment]
@willykaram, @mirie on http://leadsponsor-nyccamp2014.gotpantheon.com/  the 'Partners Spotlight' block is not on the page. Is this what we want?

**Comment by willykaram** [comment]
@rabellamy Yes. This issue was not about the general 'Partners Spotlight'. It was about getting Lead Sponsors in the footer of the page.

**Comment by rabellamy** [comment]
@willykaram, how are we looking now? http://leadsponsor-nyccamp2014.gotpantheon.com/

**Comment by willykaram** [comment]
@rabellamy y, thanks that works for now. Can you please implement on live if you have time? This is needed pretty urgently.

**Comment by likewhoa** [comment]
I am not sure I like http://leadsponsor-nyccamp2014.gotpantheon.com/partners it's not adjusting to the whole screen. See my comments in #240

**Comment by willykaram** [comment]
Please note that this issue isn't closed. We really do still need a static footer block at bottom of the page that persistently displays the Lead Partner logos.

**Comment by likewhoa** [comment]
We already have a static lead sponsor footer block, so I guess this issue can be closed now.

**Comment by willykaram** [comment]
@likewhoa No, please re-read the issue above. We still have not implemented this.
